content/id/docs/reference/glossary/extensions.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Ekstensi
+id: Extensions
+date: 2019-02-01
+full_link: /id/docs/concepts/extend-kubernetes/extend-cluster/#perluasan
+short_description: >
+  Ekstensi adalah komponen perangkat lunak yang memperluas dan terintegrasi secara mendalam dengan Kubernetes untuk mendukung perangkat keras baru.
+aka:
+tags:
+- fundamental
+- extension
+---
+Ekstensi adalah komponen perangkat lunak yang memperluas dan terintegrasi secara mendalam dengan Kubernetes untuk mendukung perangkat keras baru.
+
+<!--more-->
+
+Sebagian besar admin klaster akan menggunakan instans Kubernetes yang dihos atau didistribusikan. Akibatnya, hampir semua pengguna Kubernetes perlu menginstal [ekstensi](/id/docs/concepts/extend-kubernetes/extend-cluster/#perluasan) dan sedikit pengguna yang perlu membuat ekstensi baru.
```

Comment:
Same with PR #24427. We need to keep “instance”.
nit! “didistribusikan” is not good translation for “distribution”.
```suggestion
Sebagian besar admin klaster akan menggunakan _instance_ Kubernetes yang dihoskan (_hosted_) atau didistribusikan. Akibatnya, hampir semua pengguna Kubernetes perlu menginstal [ekstensi](/id/docs/concepts/extend-kubernetes/extend-cluster/#perluasan) dan sedikit pengguna yang perlu membuat ekstensi baru.
```

Reply:
hmm, it's based on RSNI https://github.com/jk8s/sig-docs-id/blob/master/resources/RSNI-glossarium.pdf, don't we refer to this doc too for translation?